concent_api/utils/tests/test_api_view.py

```diff
@@ -243,7 +242,7 @@ def test_api_view_should_not_rollback_changes_on_200_response(self):
         force_report_computed_task.report_computed_task                 = message.tasks.ReportComputedTask()
         force_report_computed_task.report_computed_task.task_to_compute = task_to_compute
 
-        with mock.patch('utils.logging.log_message_received', _log_message_received_200_mock):
+        with mock.patch('core.views.logging.log_message_received', _log_message_received_correct_response_mock):
```

Comment:
please fix path also in:
`test_api_view_should_rollback_changes_on_400_error'
`test_api_view_should_rollback_changes_on_500_error`